src/nvim/testdir/test_quickfix.vim

```diff
@@ -2225,3 +2225,20 @@ func Test_cclose_in_autocmd()
   augroup! QF_Test
   " call test_override('starting', 0)
 endfunc
+
+func Test_resize_from_copen()
+    augroup QF_Test
+	au!
+        au FileType qf resize 5
+    augroup END
+    try
+	" This should succeed without any exception.  No other buffers are
+	" involved in the autocmd.
+	copen
+    finally
+	augroup QF_Test
+	    au!
+	augroup END
+	augroup! QF_Test
+    endtry
```

Comment:
the indentation seems different, but I assume this is just a GitHub display bug.